From f8607fc098e1f73e08003fa243c930b7d113a7fb Mon Sep 17 00:00:00 2001
From: Moein Banihashemian <moein.banihashemian@gmail.com>
Date: Sun, 9 Nov 2014 20:22:04 +0100
Subject: [PATCH] feature3

---
 magic8/ViewController.m | 1 +
 1 file changed, 1 insertion(+)

diff --git a/magic8/ViewController.m b/magic8/ViewController.m
index d53136f..2deb082 100644
--- a/magic8/ViewController.m
+++ b/magic8/ViewController.m
@@ -56,6 +56,7 @@ NSMutableArray* magicStrings;
     magicStrings[20]=@"not bad";
     magicStrings[21]=@"not good";
     magicStrings[22]=@"feature2";
+    magicStrings[23]=@"feature3";
 }
 
 @end
-- 
1.8.4.2

